bridge/server/user/session.ts

```diff
@@ -127,7 +132,7 @@ function sessionRouter(app: Express)  {
 
 export { sessionRouter };
 export { isAuthenticated };
-export { setAuthenticatedSession as sessionAuthentication };
+export { authenticateSession as authenticateSession };
```

Comment:
Remove `as authenticateSession`. Renaming is not needed if they are the same.

Reply:
Perfect thanks :)